src/main/java/frc/robot/Subsystems/ClimberWinch.java

```diff
@@ -35,17 +35,17 @@
 
```

Comment:
For some reason Github doesn't let me comment this above line 34, but instances should be below fields (RapidReact2022 convention Code Conventions/1)

Also add a semicolon after the last Level constant (convention A.IV.a.6)